gc/base/standard/Scavenger.cpp

```diff
@@ -348,15 +348,16 @@ MM_Scavenger::collectorShutdown(MM_GCExtensionsBase* extensions)
 {
 #if defined(OMR_GC_CONCURRENT_SCAVENGER)
 	if (IS_CONCURRENT_ENABLED) {
+
+		_mainGCThread.shutdown();
+
 		/*
 		 * completeConcurrentCycle(env) is normally invoked to stop a CS cycle.
```

Comment:
adding GC threads in the mix and rewording a bit:

While we could try completing potentially active concurrent cycle (by calling  completeConcurrentCycle()), at this point both mutator and GC threads should be down and it's just simpler and sufficient to change the concurrent phase to idle.